db/document.go

```diff
@@ -156,10 +156,11 @@ type Document struct {
 	ID       string `json:"-"` // Doc id.  (We're already using a custom MarshalJSON for *document that's based on body, so the json:"-" probably isn't needed here)
 	Cas      uint64 // Document cas
 
-	Deleted        bool
-	DocExpiry      uint32
-	RevID          string
-	DocAttachments AttachmentsMeta
+	Deleted           bool
+	DocExpiry         uint32
+	RevID             string
+	DocAttachments    AttachmentsMeta
+	migrationRequired bool
```

Comment:
I feel like it would be preferable to name this property something like 'inlineSyncData'.  It's less ambiguous about what information it's providing about the nature of the document.

Reply:
Ah yeah, had that discussion but forgot!